### ISSUE TITLE
✔️ 🐞 Bug Report: Responsive issue with "loop grid" when "additional breakpoints" are activated - [ED-9129]

### ISSUE THREAD

**Issue by SiteRider** [elementor/elementor#20670]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

If additional breakpoints under experiments are activated and one or more new breakpoints are created, the items of the loop grid overlap and cover each other within the viewports of the new breakpoints. The error only occurs if a different number of columns in the loop gird for the different sizes is set.

### Steps to reproduce

1) Activate the experiment additional breakpoints
2) Create a loop grid with a custom template
3) Set a different number of columns for each size

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.39-42
	PHP Version: 7.4.33
	PHP Memory Limit: 768M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://staging23.siterider.de
	Home URL: https://staging23.siterider.de
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 512M
	Max Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: de-DE
	Timezone: 2
	Admin Email: info@siterider.de
	Debug Mode: Inactive

== Theme ==
	Name: SiteRider
	Version: 1.0.1
	Author: Dominik Rajdl
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: de_DE
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.7.5
		Author: Elementor.com

	Elementor Pro
		Version: 3.9.0
		Author: Elementor.com

	Health Check & Troubleshooting
		Version: 1.5.1
		Author: The WordPress.org community

	JetEngine
		Version: 3.0.7
		Author: Crocoblock


== Must-Use Plugins ==
	Health Check Troubleshooting Mode
		Version: 1.8.1
		Author: 


== Elementor-Experimente ==
	Optimierte DOM Ausgabe: Inaktiv
	Verbessertes Laden von Assets: Inaktiv
	Verbessertes Laden von CSS: Inaktiv
	Inline-Schriftarten-Symbole: Inaktiv
	Verbesserungen der Zugänglichkeit: Inaktiv
	Zusätzliche benutzerdefinierte Breakpoints: Aktiv
	Import Export Template Kit: Inaktiv
	Native WordPress-Widgets aus den Suchergebnissen ausblenden: Inaktiv
	admin_menu_rearrangement: Standardmäßig deaktiviert
	Flexbox Container: Aktiv
	Default to New Theme Builder: Inaktiv
	Hello Theme Header & Footer: Inaktiv
	Startseiten: Inaktiv
	Farbpalette: Inaktiv
	Bevorzugte Widgets: Inaktiv
	Admin Leiste: Inaktiv
	Page Transitions: Inaktiv
	Notes: Inaktiv
	Loop: Aktiv
	Form Submissions: Inaktiv
	Scroll Snap: Inaktiv


== Protokoll ==
	
JS: showing 20 of 28JS: 2022-11-22 12:24:05 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c15240widgetsticky\' 
JS: 2022-11-22 12:24:05 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c15248widgetsticky\' 
JS: 2022-11-22 12:24:05 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c15257widgetsticky\' 
JS: 2022-11-22 12:24:05 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c15261widgetsticky\' 
JS: 2022-11-22 12:24:05 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c15270widgetsticky\' 
JS: 2022-11-22 12:24:05 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c15276widgetsticky\' 
JS: 2022-11-22 12:24:05 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c15295widgetsticky\' 
JS: 2022-11-22 12:24:05 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c15299widgetsticky\' 
JS: 2022-11-22 12:24:34 [error X 15][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c58widgetsticky\' 
JS: 2022-11-22 12:24:34 [error X 8][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c67widgetsticky\' 
JS: 2022-11-22 12:24:34 [error X 18][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c86widgetsticky\' 
JS: 2022-11-22 12:24:35 [error X 3][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c46widgetsticky\' 
JS: 2022-11-22 12:28:21 [error X 39][https://siterider.de/wp-admin/load-scripts.php?c=0&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51824] Cannot read properties of undefined (reading \'replace\') 
JS: 2022-11-22 15:49:21 [error X 9][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c50widgetsticky\' 
JS: 2022-11-22 15:49:26 [error X 2][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c105widgetsticky\' 
JS: 2022-11-25 03:13:35 [error X 9][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c37widgetsticky\' 
JS: 2022-11-25 03:13:35 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c71widgetsticky\' 
JS: 2022-11-25 03:13:35 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c80widgetsticky\' 
JS: 2022-11-25 03:13:35 [error X 1][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c109widgetsticky\' 
JS: 2022-12-08 17:20:39 [error X 9][https://siterider.de/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:40879] Cannot create property \'guid\' on string \'c81518widgetsticky\' 

PHP: showing 4 of 4PHP: 2022-11-22 17:49:20 [notice X 4][/home/customer/www/siterider.de/public_html/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-28 19:11:00 [notice X 1][/home/customer/www/siterider.de/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-12-06 04:11:01 [notice X 73][/home/customer/www/siterider.de/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php::1430] Undefined index: _flex_size_laptop [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-12-08 18:10:13 [notice X 2][/home/customer/www/siterider.de/public_html/wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::219] Trying to get property 'ID' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 10 of 102022-12-06 23:06:24 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]
2022-12-06 23:06:25 [info] elementor::elementor_updater Started 
2022-12-06 23:06:25 [info] Elementor/Upgrades - _on_each_version Start  
2022-12-06 23:06:25 [info] Elementor/Upgrades - _on_each_version Finished 
2022-12-06 23:06:25 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]
2022-12-08 20:55:21 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]
2022-12-08 20:55:22 [info] elementor-pro::elementor_pro_updater Started 
2022-12-08 20:55:22 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-12-08 20:55:22 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-12-08 20:55:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Kompatibel

== Elementor Pro - Compatibility Tag ==
	

				
````

</details>

**Comment by nicholaszein** [comment]
Hello @SiteRider,

thank you for submitting your issue! 🙏

Can you please confirm you have tested this with the latest versions of both Elementor (v3.9.0), and Elementor Pro (v3.9.0)? I'm seeing in your System Info that you have Elementor v3.7.5, which is an older version of Elementor.

I can also see that you have JetEngine installed. Can you confirm this issue also happens without JetEngine enabled?

Thanks in advance!

**Comment by nicholaszein** [comment]
Thanks for the feedback @johnny75710! 🙏 

That was very helpful! 💪 

We are looking into it and will update you when we have more information to share.

Kind regards

**Comment by johnny75710** [comment]
So glad to help!
My apologize about the custom breakpoint. I'm facing the same issue with all the custom breakpoints, not only with the big tablet one.

Good luck!! 
Grid Loop is an amazing feature. Keep rocking guys!

**Comment by nicholaszein** [comment]
Hello @johnny75710!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.11**! 🥳 

✔️ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
